src/Client/components/ques/written-ques.tsx

```
@@ -294,6 +257,7 @@ export default function Component({
                     </div>
                 ))}
                 <div className="flex w-full justify-between items-center my-3 p-5">
+                  <div></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)